test/tests/mat4_write.at

```diff
@@ -138,7 +138,7 @@ AT_CHECK([cp $srcdir/results/write_complex_2d_numeric-double.out expout
           $builddir/test_mat readvar test_write_complex_2d_numeric.mat a],[0],[expout],[])
 AT_SKIP_IF([test -z "$MATLABEXE"])
 AT_CHECK([cp $srcdir/matlab/test_write_complex_2d_numeric.m .
-          $MATLABEXE -nosplash -nojvm -r "type='double';test_write_complex_2d_numeric" | $GREP PASSED],[0],[PASSED
```

Comment:
@massich can you explain why you needed these changes in this file?

Reply:
ok apparently one commit here from @tbeu should not be there.

Reply:
No, I cannot explain it. I am not aware to had change any `.at` file.

Reply:
Sorry, I did push-force to remove that commit again. Yes, I know, I can use branches for such uncertain commits.